eng/global.bicep

```diff
@@ -0,0 +1,2 @@
+/** The isolated subscription for testing */
+output TMESubId string = '4d042dc6-fe17-4698-a23f-ec6a8d1e98f4'
```

Comment:
I think it may be better for us to pass in a bicep/arm parameter from the test resources script like `tme_tenant: bool`. This way we don't hardcode environment IDs that are subject to change in the future.

Reply:
Addressed.

Reply:
Also to be clear, I'll move the changes under eng/common to the sdk-tools repo once we're ok with the changes here.

Reply:
@deyaaeldeen I can add an update to eng/common tomorrow. I plan to do this in a more flexible way (checking if the tenant is TME-homed).